<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchCreationDialog</class>
 <widget class="QDialog" name="BatchCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select the Actions to Include in the Batch:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_after">
     <property name="text">
      <string>Actions After Selected Container</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_before">
     <property name="text">
      <string>Actions Before Selected Container</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_imports">
     <property name="text">
      <string>Import Actions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_importStates">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Import Action State (e.g. specific file name)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
